### PR TITLE
Update 03-casestudies.Rmd

### PR DESCRIPTION
**Pull request by jymhe120**
4.9.5 Updated the NYC tree case with the interacted graph. #930 
NY_trees

### CODE REVIEW

**Comment by Cindyli0310** [comment]
@jymhe120 Thanks a lot for revising the graph. It looks much clear and easier to understand!

File: 03-casestudies.Rmd [hunk 1]
```diff
@@ -375,7 +375,7 @@ Using data from NYC Open Data, this interactive visualization shows the variety
 
 We can see what some of the familiar and uncommon trees planted in the five boroughs of New York City are. This visualization allows one to see the distribution quickly. One can make inferences based on the distribution, such as trees in the Bronx and Manhattan seem to be distributed more uniformly compared to the other three boroughs. It gives a direct comparison between the five boroughs which could be used to make a compelling decision by the audience.
 
-![NYC Street Trees](images/NY_trees.png)
+![NYC Street Trees](https://github.com/jymhe120/datasciencecoursera/blob/master/NY_trees.png)
```
Comment (gauravhassija) [inline 03-casestudies.Rmd]: I think the link for NY_trees.png should be from the reader, as in, you just update the file in the images folder and the link remains as 'images/NY_trees.png'
Don't upload it to your GitHub account and then link it.

**Comment by jymhe120** [comment]
> I think the link for NY_trees.png should be from the reader, as in, you just update the file in the images folder and the link remains as 'images/NY_trees.png'
> Don't upload it to your GitHub account and then link it.

Hi. Thank you for checking my update. However, I am not really sure what you mean. Could you please explain what is the issue with linking the image from my GitHub account.

**Comment by gauravhassija** [comment]
Having all the images in one place makes it consistent. If any personal accounts are deleted or suspended, the images will be broken.

**Pull request merged**